src/test/java/core/basesyntax/serviceimpl/ParserServiceImplTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.Transaction;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ParserServiceImplTest {
+    private static ParserServiceImpl transactionParserService;

```

Comment:
```suggestion
    private static ParserService transactionParserService;
```
use abstraction as type in every test class